sky/backends/backend_utils.py

```diff
@@ -1136,25 +1137,48 @@ def _get_tpu_vm_pod_ips(ray_config: Dict[str, Any]) -> List[str]:
     query_cmd = (f'gcloud compute tpus tpu-vm list --filter='
                  f'\\(labels.ray-cluster-name={cluster_name}\\) '
                  f'--zone={zone} --format=value\\(name\\)')
-    tpuvm_cmd = (f'gcloud compute tpus tpu-vm describe $({query_cmd})'
-                 f' --zone {zone} --format="value[delimiter=\'\\n\']'
-                 '(networkEndpoints.accessConfig.externalIp)"')
-
-    rcode, stdout, stderr = log_lib.run_with_log(tpuvm_cmd,
+    rcode, stdout, stderr = log_lib.run_with_log(query_cmd,
```

Comment:
nit: Seems like we have mostly used `returncode` instead of `rcode` as the variable name.

Reply:
Fixed.